src/app/navigation/components/NavigationProvider.spec.tsx

```diff
@@ -1,16 +1,10 @@
 import React from 'react';
 import { Provider } from 'react-redux';
 import renderer from 'react-test-renderer';
-import { createStore } from 'redux';
-import { AppState } from '../../types';
+import createTestStore from '../../../test/createTestStore';
 import NavigationProvider from './NavigationProvider';
 
-const initialState = {
-  hash: '',
-  pathname: '',
-  search: '',
-  state: {},
-};
+const initialState = new URL('https://localhost');
```

Comment:
Why is `new URL` used as `initialState`? 
Its properties are not enumerable and It passes TS validation because its `any`. If it's only for avoiding validation then it could be done in a clearer way.

Reply:
URL has the same props as Location, so it was just easy to mock that way. i didn't realize i was getting an `any` type though, and when i fix it to `Location` i realize i need to add something to match the reducer

Reply:
Spread operator only copies objects' own enumerable properties, which instances of URL inherit from its prototype. 
That is unless I'm understanding it wrong, but i've checked the state in tests and there is nothing there besides `query` and `pathname` which could lead to weird bugs.

Reply:
i updated to assign onto the url, i think that will work better

Reply:
I mean, it works up until you spread it again in `createTestStore` which does the same things as before. You would need to assign it with the pathname too.
I thought if one was to change the prototype of an object after assigning it then it wouldn't be able to tell which ones were inherited but changing prototype with `setPrototypeOf` drops inherited properties and I can't find information why.

Reply:
ok yea whatever i just hardcoded the values, in this case that is all i needed since i'm not using any of the modification functionality